### ISSUE TITLE
参数量和 FLOPs 似乎被严重低估

### ISSUE THREAD

**Issue by dieremore** [MingTian99/ESDNet#11]
作者您好，首先感谢您开源这项优秀的工作，我最近在读您这篇文章和代码的时候发现您所提供的参数量和 **FLOPs** 统计方法似乎不会把 spikingjelly 库的操作比如 `layer.Conv2d`、`layer.MultiDimensionalAttention` 纳入统计，导致这两项指标被严重低估。

在参数量上，您所提供的 thop 统计方法得到的结果是 0.165911M，但是如果使用 PyTorch 的原生统计方法 `sum(p.numel() for p in model.parameters() if p.requires_grad)` 得到的参数量是 12.811463 M，两者相差了 **70** 多倍。并且粗略估计，一个 `Spiking_Residual_Block(dim=48)` 有三个 3 * 3 卷积，`3 × 48 × 48 × 3 × 3 = 62,208 参数 ≈ 0.062M`，encoder_level1 有 4 个 Spiking_Residual_Block，仅这部分就大约有 `4 × 0.062M = 0.248M`，已经超过了您所报告的 0.165M。

在FLOPs上，我使用
```
for name, m in model.named_modules():
    if "Conv" in type(m).__name__:
        ops = getattr(m, "total_ops", None)
        print(name, type(m), "ops:", ops)
```
查看所统计的卷积操作 FLOPs，发现所有的 spikingjelly 卷积模块 layer.Conv2d 的 FLOPs 都是 0：

<img width="1261" height="71" alt="Image" src="https://github.com/user-attachments/assets/657afcf5-8b2e-48d6-aa7d-bdbb951cc898" />

<img width="1241" height="70" alt="Image" src="https://github.com/user-attachments/assets/242092a0-689f-46d0-ac6a-00a9fe207269" />  
说明占整个模型大部分计算量的脉冲卷积操作没有纳入 FLOPs 的统计范畴。同样的粗略估计，如果按照卷积 FLOPs =Hout × Wout × Cout × (Cin / groups) × Kh × Kw，仅考虑 encoder_level1 中 Spiking_Residual_Block 的卷积模块：

- encoder_level1 中有 4个 Spiking_Residual_Block 
- 每个 Spiking_Residual_Block 中有 3 个 3×3 卷积
- 输入特征尺寸 = [1, 48, 256, 256]

那么encoder_level1 中光是 Spiking_Residual_Block  的卷积 FLOPs 就有：256 × 256 × 48 × 48 × 3 × 3 × 3 × 4  ≈ 16.31 G，已经远远大于论文所报告的7.320 G，并且这还是在忽略时间步的统计结果。

除此之外，我还对论文中能耗估计的部分感到困惑。

1. 在原文中，您提到只有第一层卷积编码层（RB块）和最后一层卷积编码层均矩阵乘法运算，其他模块都只涉及 SOP。但是很显然，所提模型只有 SCU 理论上能使用纯加法算，因为只有这个模块的输入是 0/1 脉冲；而其他的模块比如 MAU 、SRB 中除了 SCU 支路的剩余部分、FRB 等都是纯粹的浮点矩阵乘法运算，完全不能使用 SOP 来估算能耗。
2. 在表1所报告的能耗中，所有的能耗计算，包括您的方法，计算的方式都是 FLOPs × 12.5 pJ，并没有使用您所提到的 SOP 计算。并且由于 FLOPs 可能被低估，能耗也存在被低估的风险。

希望作者能解答我的疑惑
